_docs/node-operator-requirements.md

```diff
@@ -92,14 +92,14 @@ Follow [these instructions](https://docs.substrate.io/v3/tools/subkey/#generatin
 oak-collator \
   --name=YOUR_COLLATOR_NAME \
   --base-path=PATH_TO_DATA_DIR \
-  --chain=PATH_TO_CHAIN_SPEC_FILE \
+  --chain=turing \
   --node-key=NODE_KEY \
   --collator \
   --force-authoring \
   --execution=wasm \
   --state-cache-size=0 \
   -- \
-  --chain=PATH_TO_RELAY_CHAIN_SPEC \
+  --chain=kusama \
```

Comment:
This probably works, but I've not tested it. For turing the relay chain is inferred from the parachain chain spec so you can leave this argument off entirely.